framework/src/test/scala/skinny/controller/feature/EnvFeatureSpec.scala

```diff
@@ -16,4 +16,10 @@ class EnvFeatureSpec extends FlatSpec with Matchers {
     }
   }
 
+  it should "Scalatra default value should be converted to lower case" in {
```

Comment:
it should "convert the default value of Scalatra's environment to lower cased one"